modules/distribution/scripts/wso2cep-samples.bat

```diff
@@ -65,6 +65,13 @@ if exist ..\repository\deployment\server\webapps\outputui.war ( copy ..\reposito
 if exist ..\repository\deployment\server\webapps\shindig.war ( copy ..\repository\deployment\server\webapps\shindig.war ..\samples\artifacts\%cn%\webapps\ > nul )
 if exist ..\repository\deployment\server\webapps\STRATOS_ROOT ( xcopy ..\repository\deployment\server\webapps\STRATOS_ROOT ..\samples\artifacts\%cn%\webapps\STRATOS_ROOT /s /q /i > nul ) )
 
+if exist ..\samples\artifacts\%cn%\webapps ( if not exist ..\samples\artifacts\%cn%\webapps\inputwebsocket.war ( copy ..\repository\deployment\server\webapps\inputwebsocket.war ..\samples\artifacts\%cn%\webapps\ > nul )
+if exist ..\samples\artifacts\%cn%\webapps ( if not exist ..\samples\artifacts\%cn%\webapps\inputwebsocket.war ( copy ..\repository\deployment\server\webapps\inputwebsocket.war ..\samples\artifacts\%cn%\webapps\ > nul )

```

Comment:
Looks like this line is duplicated. Also why are we checking  for if exist of artifacts\%cn%\webapps directory when we have checked it earlier and created the directory as well (in Line61)?